### PR TITLE
[FIX] mail_tracking: Discuss failed messages: Button and domain

### PR DESCRIPTION
**Pull request by Tardo**
Fixes the domain to found the active user in the message destinations.

It also fixes the button visibility.

cc @tecnativa

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]
Is the button being hidden on the other sections?

**Comment by Tardo** [comment]
@pedrobaeza yes

**Comment by pedrobaeza** [comment]
@Tardo please check Travis

File: mail_tracking/models/mail_message.py [hunk 1]
```diff
@@ -25,27 +25,40 @@ class MailMessage(models.Model):
...
             message.is_failed_message = bool(
                 needs_action and involves_me and has_failed_trackings)
 
+    def _search_is_failed_message(self, operator, value):
+        """Search for messages considered failed for the active user.
+        Be notice that 'notificacion_ids' is a record that change if
+        the user mark the message as readed.
+        """
+        return [
+            ('mail_tracking_ids.state', 'in', list(self.get_failed_states())),
```
Comment (chienandalu) [inline mail_tracking/models/mail_message.py]: Shouldn't you have to use the `operator` and the `value` to switch this condition betwen `in` and `not in`

**Comment by yajo** [comment]
Tests are failing.

**Comment by pedrobaeza** [comment]
@Tardo please fix this that is breaking our internal CIs.

**Comment by Tardo** [comment]
Changes done!

**Comment by pedrobaeza** [comment]
Let's fix it this finally:

/ocabot merge patch

Please fw-port it to v13.

**Pull request merged**